libtonode-tests/tests/concrete.rs

```diff
@@ -1604,10 +1598,7 @@ mod slow {
             - external_transparent_3
             - (5 * u64::from(MINIMUM_FEE));
         assert_eq!(
-            recipient
-                .wallet
-                .shielded_balance::<OrchardDomain>(None, &[])
-                .await,
+            recipient.wallet.spendable_balance::<OrchardDomain>().await,
```

Comment:
`shielded_balance` is renamed to `get_filtered_balance` it's an internal private helper that more pub interface functions call with various filter sets.  Since it's not `pub` it's not accessible in tests, and where it was called there should now be fns that acquire balances for specific sets of Outputs.